anda/tools/arduino-app-lab-bin/cc.arduino.AppLab.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Arduino App Lab
+Comment=A powerful visual environment for managing the Arduino UNO Q
+Exec=arduino-app-lab-bin
+Icon=/usr/share/pixmaps/round-arduino-logo.svg
```

This can be simplified to just this:

```suggestion
Icon=round-arduino-logo
```

---

i don't think it'll show up as an icon if registered in pixmaps

---

It will, but you should ideally just put the svg in `/usr/share/icons/hicolor/scalable/apps/arduino-app-lab` anyway

---

It does, pixmaps is just outdated is all (still usable).